src/components/Dashboard/RightClick.vue

```diff
@@ -83,8 +83,8 @@ function hasTag(tag: string) {
   return torrents.value.every(torrent => torrent && torrent.tags && torrent.tags.includes(tag))
 }
 
-async function removeTags(tags: string[]) {
-  await torrentStore.removeTorrentTags(hashes.value, tags)
+async function removeAllTags() {
+  await torrentStore.removeTorrentTags(hashes.value, [])
```

Comment:
Passing an empty list won't trigger the effect, you will need to make that parameter optional all the way through the qbit provider (and its interface)

Reply:
Thanks for the feedback, just commited a fix 👍